### PR TITLE
Java support

### PR DESCRIPTION
**Pull request by zebraxxl**

### CODE REVIEW

File: java/pom.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>winium.elements.desktop</groupId>
+    <artifactId>winium.elements.desktop</artifactId>
```
Comment (13h3r) [inline java/pom.xml]: Why `artifactId` and `groupId` are same?

File: java/src/main/java/winium/elements/desktop/ComboBox.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebElement;
+import org.openqa.selenium.remote.Response;
+
+import winium.elements.desktop.extensions.ByExtensions;
+
+import java.util.HashMap;
+
+public class ComboBox extends DesktopElement {
+    public static final String COLLAPSE_COMBO_BOX = "collapseComboBox";
```
Comment (13h3r) [inline java/src/main/java/winium/elements/desktop/ComboBox.java]: Do we really need all these constants in public scope?

**Comment by NickAb** [comment]
:+1:

**Comment by nwlunatic** [comment]
:+1:

**Pull request merged**